qiskit/optimization/problems/quadratic_program.py

```diff
@@ -1090,6 +1091,63 @@ def from_ising(self,
         self.minimize(constant=offset, linear=linear_terms, quadratic=quadratic_terms)
         offset -= offset
 
+    def get_feasibility_info(self, x: Union[List[float], np.ndarray]) \
+            -> Tuple[bool, List[Variable], List[Constraint]]:
+        """Returns whether a solution is feasible or not along with the violations.
+        Args:
+            x: the input result list returned by the optimizer
```

Comment:
Can this text be improved a bit? In the OptimizationResult x is described as `the optimal value found in the optimization` the text `the input result list` seems a bit vague. Would the `a solution value, such as returned in an optimizer result` work

Reply:
Agree.